README.md

```diff
@@ -167,3 +167,17 @@ If TrackScheme is fully zoomed, a tap of `Z` unzoom fully.
 | Undo.                               | `Control-Z`.                                                 |
 | Redo.                               | `Control-shift-Z`.                                           |
 
+# More Documentation
+
+* [Mastodon intro on ImageJ.net](https://imagej.net/plugins/mastodon/)
+* [Mastodon usage guide on readthedocs](https://mastodon.readthedocs.io/en/latest/index.html)
+
+# Community
+
+* If you have questions regarding usage of Mastodon, we invite you to use
+  the [forum on imagesc](https://forum.image.sc/). You may search for Mastodon related topics using the
+  hashtag `#mastodon` or create a new topic.
+
+## Contribute
+
+Read through [how to contribute](CONTRIBUTING.md) for information on how to contribute to this project.
```

Comment:
The same. Maybe move it to the umbrella org page.

Reply:
Good suggestion.
I removed the code from here and made a pull-request to the org page.
@tinevez Please have a look at: https://github.com/mastodon-sc/.github/pull/1